docs/content/server/setup.md

```diff
@@ -0,0 +1,102 @@
+# Setting up the server
```

Comment:
Do we want to add the "advanced" set up too or will that be too confusing for users?

Reply:
I wasn’t sure about that. I thought about adding an advanced collapsible segment. Thoughts?

Reply:
@itsababseh thoughts

Reply:
What would you add in the "advanced" set up? I think this simple version is clear enough, plus Toran's video is linked here. If we get feedback that "users are confused" "or there is difficulty setting up" then it may warrant an advanced setup section.

Reply:
The advanced would include things like .env and settings and passwords/secrets